config.json

```diff
@@ -471,6 +471,18 @@
           "text_formatting"
         ]
       },
+      {
+        "slug": "reverse-string",
+        "name": "Reverse String",
+        "uuid": "33798c5a-f2ca-4e7b-9dcc-96a24ccf9f24",
+        "practices": [
+          "strings"
+        ],
+        "prerequisites": [
+          "strings"
+        ],
+        "difficulty": 2
```

Comment:
I think that's more like a 1, for a function that's in the standard lib.

Reply:
Can do. Typically, I avoid 1 on my own tracks since that's the `hello-world` difficulty, and all exercises are at least slightly more difficult than that. :)

Reply:
Haha that's a valid point, oh well, this expresses well my view on this exercise :)